docs/encyclopedia/application-message-passing.mdx

```diff
@@ -342,16 +342,20 @@ The following content applies in every SDK except the Go SDK. See below.
 #### Exceptions in Signals
 
 In Signal handlers, throw [Application Failures](/references/failures#application-failure) only for unrecoverable errors, because the entire Workflow will fail.
+Similarly, allowing a failing Activity to exhaust its retries, so that it throws an [Activity Failure](https://docs.temporal.io/references/failures#activity-failure) will cause the entire Workflow to fail. This will only happen if you change the default Activity [Retry Policy](https://docs.temporal.io/encyclopedia/retry-policies).
```

Comment:
Presumably this occurs on a timeout as well?

The wording makes it ambiguous whether this is just an example or is intended to be comprehensive.  You might try:
"For example, Activities can cause unrecoverable errors if you change their settings from the default, such as when they time out or exhaust their retries."

Reply:
Activity timeout is just another way for a retry to be triggered, so I think it's OK as-is. Added Child Workflows, but neither now nor previously is there a claim to comprehensiveness.

Reply:
I'm referring to the [schedule-to-close timeout](https://docs.temporal.io/encyclopedia/detecting-activity-failures#schedule-to-close-timeout), which doesn't result in a retry.
(also, schedule-to-start timeout, though that's rarely used)
I think the other part of my feedback applies, regardless.    "Similarly" isn't the right word since what you're describing is a particular example of the phenomenon you described in the first paragraph.